pkg/tests/apis/playlist/testdata/openapi.json

```diff
@@ -3,6 +3,17 @@
   "info": {
     "title": "playlist.grafana.app/v0alpha1"
   },
+  "servers": [
+    {
+      "url": "/apis/playlist.grafana.app/v0alpha1/namespaces/{namespace}/",
+      "variables": {
+        "namespace": {
+          "default": "default",
+          "description": "object name and auth scope, such as for teams and projects",
+        }
+      }
+    }
+  ],
```

Comment:
So this abstracts the base URL, so we don't need to define those separately for each endpoint?

Reply:
Yes, that's the role of it in OpenAPI specs.

But, here I've used this to avoid RTK Query generating URLs that require a namespace parameter, and it'll be defined centrally/manually in the baseAPI.ts file. It's half hack, half what it's designed for.